packages/components/src/utils/json.ts

```diff
@@ -1,12 +1,20 @@
 export const parseJSONAttribute = <T>(attribute: T | string): T => {
-  return typeof attribute === 'string'
-    ? // input is potentially JSON parsable string, e.g. "{ 'aria-label': 'Some label' }"
-      JSON.parse(
-        attribute
-          .replace(/(?<!\\)'/g, '"') // convert single quotes to double quotes except the ones which are escaped by backslash
-          .replace(/\\(?!u0027)/g, '') // remove string escapes except the ones followed by unicode u0027
-          .replace(/[\s"]?([\w-]+)[\s"]?:/g, '"$1":') // wrap keys in double quotes
-      )
-    : // input is object, e.g. { 'aria-label': 'Some label' }
-      attribute;
+  // Input is object, e.g. { 'aria-label': 'Some label' }
+  if (typeof attribute !== 'string') {
+    return attribute;
```

Comment:
Isn't a single return per function preferred over multiple returns?

Reply:
This adds alot to readability since the ternary function was complex to read and understand. Also I never heard of a single return policy for functions, but feel free to lighten me up :)

Reply:
I don't think there's a rule or something. I'm also fine with multiple exit points. Maybe here's an article which prefers single returns: https://stschindler.medium.com/having-only-one-return-statement-is-better-and-more-safe-5cd379b7e87c

Reply:
Unfortunately it's behind a "registration wall" :(